src/focus.service.ts

```diff
@@ -521,11 +521,10 @@ export class FocusService {
       return false;
     }
 
-    if (el.tabIndex > -1) {
-      return true;
+    if (el.tabIndex < 0) {
+      return false;
     }
```

Comment:
My question is, if they aren't meant to be selectable, why do they have z-indicies?